src/lib/y2storage/proposal/autoinst_nfs_planner.rb

```diff
@@ -187,25 +206,29 @@ def mandatory_values(format)
       # Name of the server from a NFS share
       #
       # @param share [String] e.g., "192.168.56.1:/root_fs"
-      # @return [String]
+      # @return [String] e.g., "192.168.56.1"
       def server(share)
         server_and_path(share).first || ""
       end
 
       # Name of the shared directory from a NFS share
       #
       # @param share [String] e.g., "192.168.56.1:/root_fs"
-      # @return [String]
+      # @return [String] e.g., "/root_fs"
       def path(share)
         server_and_path(share).last || ""
```

Comment:
hmm, if path is not defined I think path is according to that RFC ".", not?

Reply:
I am not sure, but anyway, libstorage-ng does not support NFS without a path, so maybe we need another card for that.

Reply:
yep, please create card for it.